warehouse/models/mart/gtfs/fct_hourly_rt_feed_files_success.sql

```diff
@@ -34,6 +34,26 @@ parse_outcomes AS (
     {% endif %}
 ),
 
+daily_totals AS (
```

Comment:
for this one, I think you can definitely lean on the columns in [`fct_daily_rt_feed_files`](https://github.com/cal-itp/data-infra/blob/main/warehouse/models/mart/gtfs/fct_daily_rt_feed_files.sql) and construct the proportion from that rather than grouping again